onadata/apps/main/management/commands/remove_kpi_tables.py

```diff
@@ -0,0 +1,81 @@
+from django.db import connection
+from django.core.management.base import BaseCommand
+from django.utils.translation import ugettext_lazy
+
+
+def drop_table(table):
+    cursor = connection.cursor()
+    # Using DROP instead od Delete to remove entire table from DB
+    # Calling cascade to delete data referenced from other tables
+    sql = f"DROP TABLE {table} CASCADE;"
+    cursor.execute(sql)
+
+
+class Command(BaseCommand):
+    help = ugettext_lazy("Drop Formbuilder tables and data from Onadata DB")
+
+    def handle(self, *args, **kwargs):
+        kpi_tables = [
+            'django_migrations',
+            'django_content_type',
+            'auth_user',
+            'auth_group',
+            'auth_permission',
+            'auth_group_permissions',
+            'auth_user_groups',
+            'auth_user_user_permissions',
```

Comment:
Although these auth tables they were required when doing the data migration, I'm weary on droping the entire table.
There could be records referenced while dropping the other tables, but I believe we would be cascade deleting those specific records

Reply:
I'll commit out the auth tables.

Reply:
So the way the table names work is that the app name is included e.g. `auth_user` means that this is the User model from the auth app.

You can use that scheme to only target tables from the kpi app (and any other specific app).